src/components/Settings/sections/VoiceLibrarySection.scss

```
@@ -235,11 +235,42 @@
   cursor: pointer;
 
   &:hover:not(:disabled) { color: #fff; border-color: #888; }
-  &:disabled { opacity: 0.4; cursor: not-allowed; }
+  &:disabled { cursor: not-allowed; }
+  // Dim on disabled EXCEPT while showing the loading spinner. Opacity is a
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)